settings.gradle

```diff
@@ -1,3 +1,23 @@
 rootProject.name = 'apoc'
 
-include("extended")
\ No newline at end of file
+include("extended")
+
+include ':core', ':common', ':processor', ':test-utils'
+project(':core').projectDir = new File(rootDir, 'apoc-core/core')
+project(':common').projectDir = new File(rootDir, 'apoc-core/common')
+project(':processor').projectDir = new File(rootDir, 'apoc-core/processor')
+project(':test-utils').projectDir = new File(rootDir, 'apoc-core/test-utils')
+
+startParameter.excludedTaskNames = [
+        ':core:build',
+        ':common:build',
+        ':processor:build',
+        ':test-utils:build',
+        ':core:test',
+        ':common:test',
+        ':processor:test',
+        ':test-utils:test',
```

Comment:
Without these the code from `core`, `common` would be tested as well, but we don't want to do that from the `extended` repo